File: Java/백준/Bronze/33191. Yalda/Yalda.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int m = in.nextInt();
        int[] a = new int[3];
        for (int i=0;i<3;i++){
            a[i] = in.nextInt();
        }
        if(a[0]<=m){
            System.out.println("Watermelon");
        }
        else if(a[1]<=m){
            System.out.println("Pomegranates");
        }
        else if(a[2]<=m){
            System.out.println("Nuts");
        }
        else{
            System.out.println("Nothing");
        }
    }
}

File: Python/백준/Bronze/33191. Yalda/Yalda.py
money = int(input())
aa = []
for i in range(3):
    aa.append(int(input()))
if (aa[0]<=money):
    print('Watermelon')
elif (aa[1]<=money):
    print('Pomegranates')
elif (aa[2]<=money):
    print('Nuts')
else:
    print('Nothing')